src/swoop/api/config.py

```diff
@@ -12,6 +13,21 @@ class Settings(BaseSettings):
     database_name: str
     database_url: str
 
+    db_min_conn_size: int = 2
+    db_max_conn_size: int = 2
+    db_max_queries: int = 50000
+    db_max_inactive_conn_lifetime: float = 300
+
+    @property
+    def reader_connection_string(self):
+        """Create reader psql connection string."""
+        return f"postgresql://{self.database_user}:{quote(self.database_pass)}@{self.database_host}:{self.database_port}/{self.database_name}"
+
+    @property
+    def writer_connection_string(self):
+        """Create writer psql connection string."""
+        return f"postgresql://{self.database_user}:{quote(self.database_pass)}@{self.database_host}:{self.database_port}/{self.database_name}"
```

Comment:
I was going to say that the url is already in the `.env` file as `DATABASE_URL` and I was wondering if it might make sense just to use that url here. But I am thinking that it might be useful to be able to override the database name, specifically in something like tests where we don't want to conflict with an existing user's manual test database.

Reply:
Yeah I am not married to this and happy to refactor it once we're happy with our .env/local/test setup.